### DISCUSSION TITLE
Client side SDK update interval

### DISCUSSION THREAD

**Discussion by linjer** [flipt-io/flipt#3174] [Q&A] [answered]
I'm trying out the client-side SDK and am just wondering what unit the `update_interval` parameter is. Is it seconds?

**Comment by GeorgeMac** [comment, marked as answer]
Yep I believe so @linjer 
https://github.com/flipt-io/flipt-client-sdks/blob/efef2cb51936532ecafee85ea49cb229d0941f0c/flipt-engine-ffi/src/lib.rs#L114

You bring up a good point, we should document that somewhere clearly.
